### PR TITLE
Creates Speaker Series Landing Page

### PR DESCRIPTION
**Pull request by C-STYR**
- Preview [HERE](https://teleport-docs-next-git-cole-series-gravitational.vercel.app/security-visionaries-2022/)
- fixes gravitational/web#2399
- Marketo form needs alteration
- Awaiting event titles, main description, short form event descriptions

### CODE REVIEW

File: pages/security-visionaries-2022/index.mdx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: Security Visionaries Speaker Series 2022
+description: Computer security is no longer about data; it's about life and property.
+headerMode: simple
+shortFooter: true
+headerDescription: Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Maecenas sed enim ut sem viverra. Rhoncus aenean vel elit scelerisque mauris pellentesque pulvinar.
```
Comment (anadeliafadeev) [inline pages/security-visionaries-2022/index.mdx:6]:
```suggestion
headerDescription: Come hear what some of the world’s foremost security researchers, practitioners and thinkers see on the security horizon for 2022.  
```

File: pages/security-visionaries-2022/index.mdx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    </SignupDescription>
+  }
+>
+  <List.Item
+    icon="card"
+    size="sm"
+    maxWidth={["100%", "80%"]}
+    title="Securing a World of Physically Capable Computers"
+  >
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
```
Comment (anadeliafadeev) [inline pages/security-visionaries-2022/index.mdx:32]:
```suggestion
Computer security is no longer about data; it's about life and property. This change makes an enormous difference, and will shake up our industry in many ways. First, data authentication and integrity will become more important than confidentiality. And second, our largely regulation-free Internet will become a thing of the past.
Soon we will no longer have a choice between government regulation and no government regulation. Our choice is between smart government regulation and stupid government regulation. Given this future, it's vital that we look back at what we've learned from past attempts to secure these systems, and forward at what technologies, laws, regulations, economic incentives, and social norms we need to secure them in the future.

```

**Review by anadeliafadeev — APPROVED** [review]

File: pages/security-visionaries-2022/index.mdx [hunk 3]
```diff
@@ -0,0 +1,53 @@
+---
+title: Security Visionaries Speaker Series 2022
```
Comment (anadeliafadeev) [inline pages/security-visionaries-2022/index.mdx:2]:
```suggestion
title: Security Visionaries 2022
```

**Review by anadeliafadeev — APPROVED** [review]

**Comment by anadeliafadeev** [comment]
For the first event:

- Add speaker details: Bruce Schneier – Security Technologist & Guru [link to https://www.leadingauthorities.com/speakers/bruce-schneier]

- Remove the date from the title, can you add it as separate line with something like: Date and time: Tuesday January 18th, 2022 at 11am PT


For the second event:
Remove dummy text and put "Coming Soon"

**Comment by ardeche07** [comment]
Looking good! 

Michael's feedback

* Need to add Speaker names
* Need to include title/abstract for Tony before promoting
* Can we increase the size of the headshots? They are the big draw. I want people to see this as vendor neutral, not the Teleport show, so use bigger pictures like if we were promoting the rolling stones as the headline band on a concert website.
* Can we increase the font size under the H1. 
* Suggest putting talk title and date on separate lines.

**Comment by ardeche07** [comment]
Oh. We are going to have at least 4 speakers.  I think we should have a +/- to expand/collapse the talk abstract. Otherwise the page is going to be very long.

**Comment by C-STYR** [comment]
@anadeliafadeev @alexwolfe @ardeche07 - worked on most of your comments today.  Need more definite sizes for images please - these are currently 240px (Michael, use cmd+shft+4 to draw a box with pixel dimensions - also snaps a screenshot).  We could also change the format so the pic and title/date were on the same line and the text was full-width underneath, but the more restructuring we do the more time it takes.  Your call. 
I didn't have much time today to work on the expandable text, just sketched out a quick prototype - try clicking on Bruce's abstract.

**Pull request merged**